examples/add_images.py

```diff
@@ -0,0 +1,25 @@
+"""
+This example shows two image layers using the add_image API
+"""
+
+import sys
+from PyQt5.QtWidgets import QApplication
+
+from skimage import data
+from skimage.color import rgb2gray
+from napari_gui import Window, Viewer
+
+
+if __name__ == '__main__':
+    # starting
+    application = QApplication(sys.argv)
+
+    # create the viewer and window
+    viewer = Viewer()
+    window = Window(viewer)
```

Comment:
Hmmm... Any way to get around the window creation? My spidey-sense tingles because the variable isn't explicitly used anywhere...

Reply:
ah glad your spidey-sense is working as I agree, this causes me concern too. It just seems all seems quite roundabout right now. This is being discussed in #108 most actively, though is relevant for #120 and was previously talked about in a couple other issues and PRs. I think we should definitely revisit this before we go too far but I'm not sure how to change it